src/stringSimilarity.js

```diff
@@ -1,17 +1,103 @@
 import escapeRegExp from 'lodash.escaperegexp';
 import matcher from 'matcher';
+import { isObject } from './helpers';
 
 export const WILDCARD_MARKER_ESCAPED = '{{\\*}}';
 export const WILDCARD_MARKER = '{{*}}';
+export const UNQUOTED_WILDCARD_PLACEHOLDER = '{{%}}';
+const searchUnquotedWildcards = new RegExp(/(([^:\s,]*)\{\{\*\}\}([^,:\s}\]]*))(?=[^,:}\]\s]*)/g);
+const searchModifiedWildcards = new RegExp(/(([^:\s,]*)\{\{%\}\}([^,\s}\]]*))(?=[^,}\]\s]*)/g);
 
```

Comment:
What do you think of changing `searchModifiedWildcards` to `searchUnquotedWildcardPlaceholders`?

Reply:
Done!